src/components/Filter.js

```diff
@@ -210,55 +210,44 @@ function Filter(props) {
         ))}
       </TextField>
       */}
-      <GSInputLabel text="Approved" />
+      <GSInputLabel text="Verification Status" />
       <TextField
         select
         value={
-          approved === undefined ? 'All' : approved === true ? 'true' : 'false'
+          approved === undefined && active === undefined
+            ? 'All'
+            : approved === false && active === true
+            ? 'Awaiting Verification'
+            : active === true && approved === true
+            ? 'Approved'
+            : 'Rejected'
         }
-        InputLabelProps={{
-          shrink: true,
-        }}
-        onChange={(e) =>
+        onChange={(e) => {
           setApproved(
             e.target.value === 'All'
               ? undefined
-              : e.target.value === 'true'
-              ? true
-              : false,
-          )
-        }
-      >
-        {['All', 'true', 'false'].map((name) => (
-          <MenuItem key={name} value={name}>
-            {name}
-          </MenuItem>
-        ))}
-      </TextField>
-      <GSInputLabel text="Rejected" />
-      <TextField
-        select
-        value={
-          active === undefined ? 'All' : active === true ? 'false' : 'true'
-        }
-        InputLabelProps={{
-          shrink: true,
-        }}
-        onChange={(e) =>
+              : e.target.value === 'Awaiting Verification' ||
```

Comment:
It's best to create constant variables containing these labels and compare those.
That reduces the risk of a typo breaking something that won't be caught by the linter or interpreter.